Users/coafile.rst

```diff
@@ -38,8 +38,95 @@ wide coafile which has the lowest priority. The ``default_coafile`` must
 lie in the coala installation directory and is valid for everyone using
 this coala installation.
 
-Setting Inheritance
--------------------
+Explicit Setting Inheritance
+----------------------------
+
+Every coafile contains one or more sections. Section names are case
+insensitive. The old implicit section inheritance syntax has been deprecated
+and has been scheduled for removeal. Instead, define section inheritance
+explicitly by naming a section in the format ``[basesection.newsection]``.
+Extra values can be appended to an inherited setting using the ``+=``
+operator.
+
+Consider the following coafile::
+
+  [all]
+  enabled = True
+  overridable = 2
```

Comment:
What is overridable, just another setting name or something critical for section inheritance?